src/integrationTest/java/com/mongodb/kafka/connect/mongodb/MongoKafkaTestCase.java

```diff
@@ -303,14 +324,52 @@ public T deserialize(final String topic, final byte[] data) {
     }
   }
 
+  public <K, V, T> List<T> getProduced(
```

Comment:
Added an extra method that just has an expected size value and not a list of expected items.  We use this to as part of the getProducedStrings helper to get all the keys from the heartbeats collection.